public/app/plugins/datasource/loki/components/LokiQueryEditor.tsx

```diff
@@ -75,7 +61,7 @@ export class LokiQueryEditor extends PureComponent<Props, State> {
           onRunQuery={this.onRunQuery}
           history={[]}
           data={data}
-          absoluteRange={this.calcAbsoluteRange(data)}
+          absoluteRange={createAbsoluteRange(data)}
```

Comment:
I will have to do a check with platform team regarding if I can pass timerange from QueryRow to Editor.